src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1355,14 +1356,15 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
 
         $classToServices = [
             LocoProviderFactory::class => 'translation.provider_factory.loco',
+            LokaliseProviderFactory::class => 'translation.provider_factory.lokalise',
         ];
 
         $parentPackages = ['symfony/framework-bundle', 'symfony/translation', 'symfony/http-client'];
 
         foreach ($classToServices as $class => $service) {
             $package = sprintf('symfony/%s-translation-provider', substr($service, \strlen('translation.provider_factory.')));
 
-            if (!$container->hasDefinition('http_client') || !ContainerBuilder::willBeAvailable($package, $class, $parentPackages)) {
+            if (!$container->hasDefinition('http_client') || !ContainerBuilder::willBeAvailable(sprintf('symfony/%s-translation', $package), $class, $parentPackages)) {
```

Comment:
```suggestion
            if (!$container->hasDefinition('http_client') || !ContainerBuilder::willBeAvailable(sprintf('symfony/%s-translation-provider', $package), $class, $parentPackages)) {
```

Please add the suffix